test/fixtures/3.0/ping/.prettierignore

```diff
@@ -0,0 +1,2 @@
+dist
+*.json
```

Comment:
Ditto. Can we remove this file? Alternatively, let's move it to the root directory.